app/src/main/res/layout/custom_confirm_dialog.xml

```diff
@@ -0,0 +1,90 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.cardview.widget.CardView xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Infact this is not needed use a previous dialog just change its text

Reply:
done